install/gce-installation-guide.md

```diff
@@ -0,0 +1,119 @@
+# Installing Kata Containers on Google Compute Engine
```

Comment:
Could you remove the gerund here (and in the other section headings) for consistency with our other docs?

/cc @klynnrif.

Reply:
Done.